repository.rosinstall

```diff
@@ -3,6 +3,11 @@
     uri: https://github.com/b-it-bots/mas_common_robotics.git
     version: kinetic
 
+- git:
```

Comment:
wouldn't mas_industrial_robotics already be installed/cloned since the .rosinstall file is in this repo?

Reply:
The mas_industrial_repo is not cloned in the catkin_ws initially. It is cloned elsewhere in order to run ./setup.sh which then creates the catkin_ws. The repositories in the rosinstall file are then added to the catkin_ws. Earlier it was manually copied over to the catkin_ws using 

`cp -r $ROOT_DIR ~/catkin_ws/src`
in setup.sh. I have removed this cp command, I have added it directly to rosinstall so that it is directly cloned into the catkin_ws along with other repos.